nav2_loopback_sim/nav2_loopback_sim/loopback_simulator.py

```diff
@@ -209,6 +230,133 @@ def debug(self, msg):
         self.get_logger().debug(msg)
         return
 
+    def getMap(self):
+        request = GetMap.Request()
+        if self.map_client.wait_for_service(timeout_sec=5.0):
+            # Request to get map
+            future = self.map_client.call_async(request)
+            rclpy.spin_until_future_complete(self, future)
+            if future.result() is not None:
+                self.map = future.result().map
+                self.get_logger().info('Laser scan will be populated')
+            else:
+                self.get_logger().warn('Failed to get map')
+                self.get_logger().info('Laser scan will be populated using max range')
+        else:
+            self.get_logger().warn('Failed to get map')
+            self.get_logger().info('Laser scan will be populated using max range')
+
+    def getLaserPose(self):
+        try:
+            if self.initial_pose is None:
+                return 0.0, 0.0, 0.0
```

Comment:
`None`s? (as well in the exception handling) That would make it the same as the map to world conversion methods error handling